test/inbox.ts

```diff
@@ -125,7 +125,7 @@ describe('Inboxes', function () {
 			response._links.should.exist;
 			response._links.self.should.startWith(`${apiUrl}/inboxes/${keys.testInboxId}/teammates`);
 			response._results.should.exist;
-			response._results.length.should.be.gt(1);
+			response._results.length.should.be.gte(1);
```

Comment:
Has something changed in the API that means this and line 112 don't hold true anymore?

Reply:
My test (it being free) instance only has one teammate.